extensions/qute/component/src/generated/resources/org/apache/camel/component/qute/qute.json

```diff
@@ -11,7 +11,7 @@
     "supportLevel": "Preview",
     "groupId": "org.apache.camel.quarkus",
     "artifactId": "camel-quarkus-qute-component",
-    "version": "2.16.0-SNAPSHOT",
+    "version": "2.17.0-SNAPSHOT",
```

Comment:
Well, I think this change should not be included.

Reply:
Thanks removed from PR!

Reply:
@svkcemk Please don't remove `qute.json` but just update the `version` in a new PR. Just curious how did you get this change, I think it should be included in the `Next is 2.17.0-SNAPSHOT` commit. see https://camel.apache.org/camel-quarkus/2.16.x/contributor-guide/release-guide.html#_next_version_in_camel_quarkus_main_branch

Reply:
@zhfeng I think its was part of the auto generated extension part, I added the quto.json back, thanks!

Reply:
@svkcemk yeah, I thinks so. Can you share the instructions you used for building? I belive `camel-package-maven-plugin` generates these files but I can not get it working by using `./mvn -f extensions/qute/component/pom.xml clean install`

Reply:
Yeah I used `mvn clean install -DskipTests` to build the complete project.

Reply:
and `qute.json` which you added back seems in a wrong directory.

Reply:
sorry about it do you know how can I re-generate that ?

Reply:
let me try to fix I think I got it

Reply:
TBH, I don't know neither and let me check.

Reply:
Thanks aw I added back to correct place.

Reply:
@zhfeng I tried to re-generate it using `mvn cq:create -N -Dcq.artifactIdBase=cli-connector` in a separate place and then added it back.

Reply:
@svkcemk I tried the following steps
```
./mvnw cq:create -N -Dcq.artifactIdBase=cli-connector -Dcq.nativeSupported=false
./mvnw clean install -DskipTests
```

But it added itest in `integration-tests/cli-connector/` and  the building failed with
```
[ERROR] Failed to execute goal org.codehaus.gmaven:groovy-maven-plugin:2.1.1:execute (validate-github-workflows) on project camel-quarkus: Execution validate-github-workflows of goal org.codehaus.gmaven:groovy-maven-plugin:2.1.1:execute failed: Integration tests not executed by the CI:
[ERROR] 
[ERROR]     cli-connector
[ERROR] 
[ERROR]  Add the missing test module(s) to tooling/scripts/test-categories.yaml

```

@ppalaga Is it a bug in cq-maven-plugin?

Reply:
> I tried the following steps
> 
> ```
> ./mvnw cq:create -N -Dcq.artifactIdBase=cli-connector -Dcq.nativeSupported=false
> ./mvnw clean install -DskipTests
> ```
> 
> But it added itest in `integration-tests/cli-connector/` and the building failed with
> 
> ```
> [ERROR] Failed to execute goal org.codehaus.gmaven:groovy-maven-plugin:2.1.1:execute (validate-github-workflows) on project camel-quarkus: Execution validate-github-workflows of goal org.codehaus.gmaven:groovy-maven-plugin:2.1.1:execute failed: Integration tests not executed by the CI:
> [ERROR] 
> [ERROR]     cli-connector
> [ERROR] 
> [ERROR]  Add the missing test module(s) to tooling/scripts/test-categories.yaml
> ```
> 
> @ppalaga Is it a bug in cq-maven-plugin?

No, I would not say it is a bug. Adding to test-categories.yaml needs to be done manually, because it is better to take the actual test execution times into consideration. The plugin does not know those numbers. In the past, I thought of storing the the CI module execution times somewhere, so that the plugin can add the new test to the currently fastest group, but I never come to it. I failed at finding some free metrics storage service.

Reply:
yeah, but we are trying to add a JVM-only extension and it should put itest in `integration-tests-jvm` ?

Reply:
@ppalaga I tried a fix on https://github.com/l2x6/cq-maven-plugin/pull/145, can you take a look?